website/docs/docs/guides/debugging-errors.md

```diff
@@ -0,0 +1,378 @@
+---
+title: Debugging errors
+---
+## General process of debugging
+Learning how to debug is a skill, and one that will make you great at your role!
+1. Read the error message — we try our best to write as much information as possible. It will normally contain the type of error, and the file where the error occurred. There's more details about different types of errors below.
```

Comment:
should we reframe this from "we" to "dbt"?

Reply:
Changed it slightly